### DISCUSSION TITLE
Deployment frequency

### DISCUSSION THREAD

**Discussion by mtweeman** [renovatebot/renovate#32645] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Hey, out of curiosity as I haven't found the answer to this question anywhere. How often do you deploy the newest version of app as shown in repository releases? Can that be seen anywhere? Is there some explicit schedule?
From logs I'm able to see, that currently it's 39.19.0.

**Comment by rarkins** [comment, marked as answer]
Usually 1-2 times per week. More frequent if there's an important bug fix
